pkgs/servers/emby/default.nix

```diff
@@ -21,26 +21,22 @@ stdenv.mkDerivation rec {
   ];
 
   propagatedBuildInputs = [
-    mono54
+    dotnet-sdk
     sqlite
   ];
 
   preferLocalBuild = true;
 
-  # Need to set sourceRoot as unpacker will complain about multiple directory output
-  sourceRoot = ".";
-
   buildPhase = ''
-    substituteInPlace SQLitePCLRaw.provider.sqlite3.dll.config --replace libsqlite3.so ${sqlite.out}/lib/libsqlite3.so
-    substituteInPlace MediaBrowser.Server.Mono.exe.config --replace ProgramData-Server "/var/lib/emby/ProgramData-Server"
+    rm -rf {electron,runtimes}
   '';
 
   installPhase = ''
-    mkdir -p "$out/bin"
-    cp -r * "$out/bin"
+    install -dm 755 "$out/usr/lib/emby-server"
+    cp -r * "$out/usr/lib/emby-server"
```

Comment:
I suspect this will copy `env-vars` to the destination directory, which can contain references to other projects not needed at runtime, similar to #47269

Reply:
What do you suggest would be appropriate?

Reply:
In the linked issue, I have changed the unpacking phase to create the expected source directory
```
unpackPhase = "tar xvzf $src";
    unpackPhase = ''
      mkdir src
      cd src
      tar xvzf $src
    '';
```

Reply:
I can't figure out what shouldn't be copied. By inspecting the result simlink it seems that only needed files are here.

Reply:
I have tested the build and there are no unwanted files in the result directory.
It seams I have misunderstood the origin of the problem in the other issue:
I assumed that the env-vars file is created as part of the setup for every build, but it is not. Also it seams to be not required, that an archive contains a directory for the files first. I will have to look at why it is again.
So I am sorry for adding this confusion.